runtime/near-vm-logic/src/context.rs

```diff
@@ -53,8 +54,8 @@ pub struct VMContext {
     #[serde(with = "crate::serde_with::bytes_as_base58")]
     /// Initial seed for randomness
     pub random_seed: Vec<u8>,
-    /// Whether the execution should not charge any costs.
-    pub is_view: bool,
+    /// If present, defines that view execution is enabled and configures it.
```

Comment:
That's a good comment -- it explains the meaning of the `None / Some`. Might make sense to unpack what view is: if non-None, we are in a view mode, which means that only read-only operations are allowed.